nntrainer/graph/graph_core.h

```diff
@@ -187,11 +204,11 @@ class GraphCore {
    * @retval    Graph Object copyed
    */
   GraphCore &copy(GraphCore &from) {
-    // if (this != &from) {
-    //   // FIXME: this assumes elements already in nodes/adj, solve that
-    //   for (unsigned int i = 0; i < adj.size(); i++)
-    //     adj[i].front()->getObject()->copy(from.adj[i].front()->getObject());
-    // }
+    if (this != &from) {
```

Comment:
I wonder if we can substitute copy to use default copy operator

Reply:
i am not sure, this is coming from network_graph directly. There are no users of copy, so will need some test in a latter PR.

Reply:
Okay